tests/integration/components/frost-bunsen-detail/arrays/array-of-booleans-test.js

```diff
@@ -0,0 +1,184 @@
+import {expect} from 'chai'
+import {describeComponent} from 'ember-mocha'
+import hbs from 'htmlbars-inline-precompile'
+import {afterEach, beforeEach, describe, it} from 'mocha'
+import sinon from 'sinon'
+
+import selectors from 'dummy/tests/helpers/selectors'
+
+describeComponent(
+  'frost-bunsen-detail',
+  'Integration: Component | frost-bunsen-detail | array of booleans',
```

Comment:
Ok, I vote to start phasing out the `|` in the test description. It makes it difficult to scope with the `filter` option.